cmd/go-mockgen/args.go

```diff
@@ -67,7 +71,7 @@ func parseFlags() (*generation.Options, error) {
 	app := kingpin.New(consts.Name, consts.Description).Version(consts.Version)
 	app.UsageWriter(os.Stdout)
 
-	app.Arg("path", "The import paths used to search for eligible interfaces").Required().StringsVar(&opts.PackageOptions[0].ImportPaths)
+	app.Arg("path", "The import paths used to search for eligible interfaces").StringsVar(&opts.PackageOptions[0].ImportPaths)
```

Comment:
This change should be covered later by https://github.com/derision-test/go-mockgen/pull/50/files#diff-bec68f6a8e7a9ffdba1264a1140ed04a177aade22c6e516c9d47cd36cd6e1656R334-R336.
It has to be non-required so that --manifest-dir can be passed in isolation